ratis-server-api/src/main/java/org/apache/ratis/server/RaftServer.java

```diff
@@ -249,5 +262,11 @@ public Builder setParameters(Parameters parameters) {
       this.parameters = parameters;
       return this;
     }
+
+    // TODO(jiacheng): add UTs
```

Comment:
If this structure looks okay to reviewers, I will add some UTs to cover the logics I added